src/camvidlog2/bioclip/cli.py

```
@@ -0,0 +1,150 @@
+            print("File already loaded")
+            print(video)
+            continue
+
+        video_path = Path(video)
+        if not video_path.exists():
+            print("file does not exist")
+            print(video)
+            continue
+
+        array = create(
+            video_path, get_video_embeddings(generate_frames_cv2(video_path))
```

Ruff (0.11.9)

Trailing comma missing

Add trailing comma

(COM812)